src/Kiota.Builder/Writers/Shell/ShellCodeMethodWriter.cs

```diff
@@ -336,6 +343,26 @@ private void WriteContainerCommand(CodeMethod codeElement, LanguageWriter writer
             writer.WriteLine("return command;");
         }
 
+        private string GetTargetTypeDeclarationRelativeToElement(CodeTypeBase targetType, CodeMethod codeElement)
+        {
+            var targetClass = conventions.GetTypeString(targetType, codeElement);
```

Comment:
this should be in the convention service, and actually probably already exists in dotnet for you to reuse.